### PR TITLE
Add A11 storage issue to faq

### PR DESCRIPTION
**Pull request by OncePunchedMan**

### CODE REVIEW

File: src/help/guides/troubleshooting.md [hunk 1]
```diff
@@ -69,6 +69,58 @@ Backup your data, uninstall the app, then restore the data in the fresh install.
...
+  * **Android 8.x**: **Settings > System > About Phone > Build Number**
+  * **Android 7.1** and lower: **Settings > About Phone > Build Number**
+
+3. Quickly tap **Build number** 7 times
+4. You will see a message appear that says "You are now a developer"
+5. Now go back and you’ll see **Developer options** listed
+6. Go back to **Settings**
+7. Open **System > Developer options** or just **Developer options**
+8. Scroll until you find **USB debugging/Android Debugging**
+9. Toggle the switch to enable it
```
Comment (ghostbear) [inline src/help/guides/troubleshooting.md:99]: How about this 
```suggestion
If you use a custom ROM or don't find the **Build number** try googling how to enable Developer Mode on your specific device

1. Open **Settings**
2. (Only on Android 8.X) Scroll down/Select to **System**
3. Select **About phone**
4. Quickly tap **Build number** 7 times
5. You will see a message appear that says "You are now a developer"
6. Now go back and you’ll see **Developer options** listed
7. Go back to **Settings**
8. Open **System > Developer options** or just **Developer options**
9. Scroll until you find **USB debugging/Android Debugging**
10. Toggle the switch to enable it
```

File: src/help/guides/troubleshooting.md [hunk 2]
```diff
@@ -68,6 +68,58 @@ Backup your data, uninstall the app, then restore the data in the fresh install.
...
+3. Open the **platform-tools** folder on your computer
+4. Shift+Right click and select **Open command prompt here** or **Open PowerShell window here**
+5. Type `adb devices` and hit Enter
+6. You should see an entry with your device(s)
+7. Now you can enter the following ADB command!
+
+
+**ADB command**
+
+`adb shell cmd appops set eu.kanade.tachiyomi android:legacy_storage allow`
```
Comment (ghostbear) [inline src/help/guides/troubleshooting.md:111]: Same here
```suggestion
`.\adb shell cmd appops set eu.kanade.tachiyomi android:legacy_storage allow`
```

**Review by ghostbear — COMMENTED** [review]
LGTM 👍 

Only thing is that the guide is a bit more geared towards Windows user

But I will let @Soitora do a final review of this

**Comment by Soitora** [comment]
Should be fine, it's a relatively similiar process on most platforms
Worst case, they can ask on Discord

**Review by Soitora — APPROVED** [review]

**Pull request merged**